src/aooptionsdialog.cpp

```diff
@@ -465,6 +465,18 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
 
   ui_gameplay_form->setWidget(row, QFormLayout::FieldRole, ui_category_stop_cb);
 
+  row += 1;
+  ui_sfx_on_idle_lbl = new QLabel(ui_form_layout_widget);
+  ui_sfx_on_idle_lbl->setText(tr("Play SFX on Idle:"));
```

Comment:
misleading label. should be "Always Send SFX" since this has to do with client to server.

Reply:
I agree